chat.js

```diff
@@ -784,7 +792,11 @@ class CommandContext {
 	 * @param {User?} [targetUser]
 	 */
 	canTalk(message = null, room = null, targetUser = null) {
-		// @ts-ignore
+		if (this.room instanceof Rooms.GlobalRoom) {
```

Comment:
This is also not equivalent to the previous code, because `this.room` _can_ be a global room as long as `room` isn't.